package.json

```diff
@@ -123,7 +123,7 @@
 		"date-fns": "^3.3.1",
 		"electron-squirrel-startup": "^1.0.0",
 		"electron2appx": "^2.1.2",
-		"express": "4.19.2",
+		"express": "4.20.0",
```

Comment:
Why not `4.21.2`?

Reply:
I upgraded to the one with a security fix in place, but it's a good point, it makes sense to upgrade to the latest 4.x. I've just upgraded it.